[2024-12-04T16:35:47.596+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-04T16:35:47.613+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sparking_flow.python_job manual__2024-12-04T16:35:45.230775+00:00 [queued]>
[2024-12-04T16:35:47.620+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sparking_flow.python_job manual__2024-12-04T16:35:45.230775+00:00 [queued]>
[2024-12-04T16:35:47.622+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-04T16:35:47.631+0000] {taskinstance.py:2889} INFO - Executing <Task(SparkSubmitOperator): python_job> on 2024-12-04 16:35:45.230775+00:00
[2024-12-04T16:35:47.639+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=185) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-04T16:35:47.637+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'sparking_flow', 'python_job', 'manual__2024-12-04T16:35:45.230775+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/spark_***.py', '--cfg-path', '/tmp/tmp_anpl7xj']
[2024-12-04T16:35:47.640+0000] {standard_task_runner.py:72} INFO - Started process 189 to run task
[2024-12-04T16:35:47.640+0000] {standard_task_runner.py:105} INFO - Job 89: Subtask python_job
[2024-12-04T16:35:47.691+0000] {task_command.py:467} INFO - Running <TaskInstance: sparking_flow.python_job manual__2024-12-04T16:35:45.230775+00:00 [running]> on host 2a892a3469ec
[2024-12-04T16:35:47.767+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Yusuf Ganiyu' AIRFLOW_CTX_DAG_ID='sparking_flow' AIRFLOW_CTX_TASK_ID='python_job' AIRFLOW_CTX_EXECUTION_DATE='2024-12-04T16:35:45.230775+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-04T16:35:45.230775+00:00'
[2024-12-04T16:35:47.768+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-04T16:35:47.789+0000] {base.py:84} INFO - Retrieving connection 'spark-conn'
[2024-12-04T16:35:47.790+0000] {spark_submit.py:473} INFO - Spark-Submit cmd: spark-submit --master spark://sparkingflow-spark-master-1:7077 --name arrow-spark --deploy-mode client jobs/python/wordcountjob.py
[2024-12-04T16:35:53.186+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:53 INFO SparkContext: Running Spark version 3.5.3
[2024-12-04T16:35:53.194+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:53 INFO SparkContext: OS info Linux, 6.10.14-linuxkit, aarch64
[2024-12-04T16:35:53.200+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:53 INFO SparkContext: Java version 17.0.13
[2024-12-04T16:35:53.305+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:53 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-12-04T16:35:53.487+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:53 INFO ResourceUtils: ==============================================================
[2024-12-04T16:35:53.490+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:53 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-12-04T16:35:53.492+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:53 INFO ResourceUtils: ==============================================================
[2024-12-04T16:35:53.494+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:53 INFO SparkContext: Submitted application: PythonWordCount
[2024-12-04T16:35:53.551+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:53 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-12-04T16:35:53.567+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:53 INFO ResourceProfile: Limiting resource is cpu
[2024-12-04T16:35:53.569+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:53 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-12-04T16:35:53.660+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:53 INFO SecurityManager: Changing view acls to: ***
[2024-12-04T16:35:53.663+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:53 INFO SecurityManager: Changing modify acls to: ***
[2024-12-04T16:35:53.669+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:53 INFO SecurityManager: Changing view acls groups to:
[2024-12-04T16:35:53.670+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:53 INFO SecurityManager: Changing modify acls groups to:
[2024-12-04T16:35:53.671+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:53 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2024-12-04T16:35:54.302+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:54 INFO Utils: Successfully started service 'sparkDriver' on port 38741.
[2024-12-04T16:35:54.526+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:54 INFO SparkEnv: Registering MapOutputTracker
[2024-12-04T16:35:54.689+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:54 INFO SparkEnv: Registering BlockManagerMaster
[2024-12-04T16:35:54.782+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:54 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-12-04T16:35:54.785+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:54 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-12-04T16:35:54.808+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:54 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-12-04T16:35:54.929+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:54 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-728275b2-8d45-4b43-ac45-3734476946ea
[2024-12-04T16:35:55.014+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:55 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-12-04T16:35:55.080+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:55 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-12-04T16:35:55.726+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:55 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-12-04T16:35:55.960+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:55 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2024-12-04T16:35:55.968+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:55 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2024-12-04T16:35:55.992+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:55 INFO Utils: Successfully started service 'SparkUI' on port 4042.
[2024-12-04T16:35:56.304+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:56 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://sparkingflow-spark-master-1:7077...
[2024-12-04T16:35:56.486+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:56 INFO TransportClientFactory: Successfully created connection to sparkingflow-spark-master-1/172.18.0.5:7077 after 106 ms (0 ms spent in bootstraps)
[2024-12-04T16:35:56.707+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:56 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20241204163556-0001
[2024-12-04T16:35:56.719+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:56 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 45221.
[2024-12-04T16:35:56.722+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:56 INFO NettyBlockTransferService: Server created on 2a892a3469ec:45221
[2024-12-04T16:35:56.723+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:56 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-12-04T16:35:56.733+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:56 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 2a892a3469ec, 45221, None)
[2024-12-04T16:35:56.738+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:56 INFO BlockManagerMasterEndpoint: Registering block manager 2a892a3469ec:45221 with 434.4 MiB RAM, BlockManagerId(driver, 2a892a3469ec, 45221, None)
[2024-12-04T16:35:56.740+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:56 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 2a892a3469ec, 45221, None)
[2024-12-04T16:35:56.742+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:56 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 2a892a3469ec, 45221, None)
[2024-12-04T16:35:57.013+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:57 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2024-12-04T16:35:57.347+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:57 INFO SparkContext: Added file /tmp/test1234.txt at spark://2a892a3469ec:38741/files/test1234.txt with timestamp 1733330157346
[2024-12-04T16:35:57.348+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:57 INFO Utils: Copying /tmp/test1234.txt to /tmp/spark-85b98228-6260-49b1-b283-09722d64b02f/userFiles-00d1bdcd-e472-4cec-849c-6623ce542e01/test1234.txt
[2024-12-04T16:35:57.382+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:57 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2024-12-04T16:35:57.385+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:57 INFO SharedState: Warehouse path is 'file:/opt/***/spark-warehouse'.
[2024-12-04T16:35:58.381+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:58 INFO InMemoryFileIndex: It took 40 ms to list leaf files for 1 paths.
[2024-12-04T16:35:58.506+0000] {spark_submit.py:634} INFO - 24/12/04 16:35:58 INFO InMemoryFileIndex: It took 2 ms to list leaf files for 1 paths.
[2024-12-04T16:36:00.647+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:00 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20241204163556-0001/0 on worker-20241204163512-172.18.0.7-41941 (172.18.0.7:41941) with 2 core(s)
[2024-12-04T16:36:00.670+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:00 INFO StandaloneSchedulerBackend: Granted executor ID app-20241204163556-0001/0 on hostPort 172.18.0.7:41941 with 2 core(s), 1024.0 MiB RAM
[2024-12-04T16:36:00.716+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:00 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20241204163556-0001/0 is now RUNNING
[2024-12-04T16:36:01.612+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:01 INFO FileSourceStrategy: Pushed Filters:
[2024-12-04T16:36:01.616+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:01 INFO FileSourceStrategy: Post-Scan Filters: (length(trim(value#0, None)) > 0)
[2024-12-04T16:36:02.405+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO CodeGenerator: Code generated in 368.403 ms
[2024-12-04T16:36:02.475+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO MemoryStore: Block broadcast_0 stored as values in memory (estimated size 199.3 KiB, free 434.2 MiB)
[2024-12-04T16:36:02.534+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO MemoryStore: Block broadcast_0_piece0 stored as bytes in memory (estimated size 34.3 KiB, free 434.2 MiB)
[2024-12-04T16:36:02.541+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO BlockManagerInfo: Added broadcast_0_piece0 in memory on 2a892a3469ec:45221 (size: 34.3 KiB, free: 434.4 MiB)
[2024-12-04T16:36:02.546+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO SparkContext: Created broadcast 0 from csv at NativeMethodAccessorImpl.java:0
[2024-12-04T16:36:02.560+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO FileSourceScanExec: Planning scan with bin packing, max size: 4194304 bytes, open cost is considered as scanning 4194304 bytes.
[2024-12-04T16:36:02.684+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO SparkContext: Starting job: csv at NativeMethodAccessorImpl.java:0
[2024-12-04T16:36:02.716+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO DAGScheduler: Got job 0 (csv at NativeMethodAccessorImpl.java:0) with 1 output partitions
[2024-12-04T16:36:02.717+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO DAGScheduler: Final stage: ResultStage 0 (csv at NativeMethodAccessorImpl.java:0)
[2024-12-04T16:36:02.718+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO DAGScheduler: Parents of final stage: List()
[2024-12-04T16:36:02.719+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO DAGScheduler: Missing parents: List()
[2024-12-04T16:36:02.722+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO DAGScheduler: Submitting ResultStage 0 (MapPartitionsRDD[3] at csv at NativeMethodAccessorImpl.java:0), which has no missing parents
[2024-12-04T16:36:02.851+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO MemoryStore: Block broadcast_1 stored as values in memory (estimated size 13.5 KiB, free 434.2 MiB)
[2024-12-04T16:36:02.872+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO MemoryStore: Block broadcast_1_piece0 stored as bytes in memory (estimated size 6.4 KiB, free 434.2 MiB)
[2024-12-04T16:36:02.875+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO BlockManagerInfo: Added broadcast_1_piece0 in memory on 2a892a3469ec:45221 (size: 6.4 KiB, free: 434.4 MiB)
[2024-12-04T16:36:02.882+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO SparkContext: Created broadcast 1 from broadcast at DAGScheduler.scala:1585
[2024-12-04T16:36:02.945+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO DAGScheduler: Submitting 1 missing tasks from ResultStage 0 (MapPartitionsRDD[3] at csv at NativeMethodAccessorImpl.java:0) (first 15 tasks are for partitions Vector(0))
[2024-12-04T16:36:02.947+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:02 INFO TaskSchedulerImpl: Adding task set 0.0 with 1 tasks resource profile 0
[2024-12-04T16:36:03.660+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:03 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Registered executor NettyRpcEndpointRef(spark-client://Executor) (172.18.0.7:58862) with ID 0,  ResourceProfileId 0
[2024-12-04T16:36:03.710+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:03 INFO BlockManagerMasterEndpoint: Registering block manager 172.18.0.7:42819 with 434.4 MiB RAM, BlockManagerId(0, 172.18.0.7, 42819, None)
[2024-12-04T16:36:03.754+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:03 INFO TaskSetManager: Starting task 0.0 in stage 0.0 (TID 0) (172.18.0.7, executor 0, partition 0, PROCESS_LOCAL, 9852 bytes)
[2024-12-04T16:36:03.999+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:03 INFO BlockManagerInfo: Added broadcast_1_piece0 in memory on 172.18.0.7:42819 (size: 6.4 KiB, free: 434.4 MiB)
[2024-12-04T16:36:04.648+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:04 INFO BlockManagerInfo: Added broadcast_0_piece0 in memory on 172.18.0.7:42819 (size: 34.3 KiB, free: 434.4 MiB)
[2024-12-04T16:36:05.020+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 WARN TaskSetManager: Lost task 0.0 in stage 0.0 (TID 0) (172.18.0.7 executor 0): org.apache.spark.SparkFileNotFoundException: File file:/tmp/spark-85b98228-6260-49b1-b283-09722d64b02f/userFiles-00d1bdcd-e472-4cec-849c-6623ce542e01/test1234.txt does not exist
[2024-12-04T16:36:05.020+0000] {spark_submit.py:634} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2024-12-04T16:36:05.020+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:781)
[2024-12-04T16:36:05.021+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2024-12-04T16:36:05.021+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2024-12-04T16:36:05.021+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2024-12-04T16:36:05.021+0000] {spark_submit.py:634} INFO - at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2024-12-04T16:36:05.022+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage1.processNext(Unknown Source)
[2024-12-04T16:36:05.022+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2024-12-04T16:36:05.022+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2024-12-04T16:36:05.022+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2024-12-04T16:36:05.022+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-12-04T16:36:05.023+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-12-04T16:36:05.023+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-12-04T16:36:05.023+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-12-04T16:36:05.023+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-12-04T16:36:05.023+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-12-04T16:36:05.023+0000] {spark_submit.py:634} INFO - at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-12-04T16:36:05.024+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-12-04T16:36:05.025+0000] {spark_submit.py:634} INFO - at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-12-04T16:36:05.025+0000] {spark_submit.py:634} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-12-04T16:36:05.025+0000] {spark_submit.py:634} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-12-04T16:36:05.025+0000] {spark_submit.py:634} INFO - at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-12-04T16:36:05.026+0000] {spark_submit.py:634} INFO - at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-12-04T16:36:05.026+0000] {spark_submit.py:634} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2024-12-04T16:36:05.026+0000] {spark_submit.py:634} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2024-12-04T16:36:05.027+0000] {spark_submit.py:634} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-12-04T16:36:05.027+0000] {spark_submit.py:634} INFO - 
[2024-12-04T16:36:05.027+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO TaskSetManager: Starting task 0.1 in stage 0.0 (TID 1) (172.18.0.7, executor 0, partition 0, PROCESS_LOCAL, 9852 bytes)
[2024-12-04T16:36:05.053+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO TaskSetManager: Lost task 0.1 in stage 0.0 (TID 1) on 172.18.0.7, executor 0: org.apache.spark.SparkFileNotFoundException (File file:/tmp/spark-85b98228-6260-49b1-b283-09722d64b02f/userFiles-00d1bdcd-e472-4cec-849c-6623ce542e01/test1234.txt does not exist
[2024-12-04T16:36:05.055+0000] {spark_submit.py:634} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2024-12-04T16:36:05.055+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO TaskSetManager: Starting task 0.2 in stage 0.0 (TID 2) (172.18.0.7, executor 0, partition 0, PROCESS_LOCAL, 9852 bytes)
[2024-12-04T16:36:05.070+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO TaskSetManager: Lost task 0.2 in stage 0.0 (TID 2) on 172.18.0.7, executor 0: org.apache.spark.SparkFileNotFoundException (File file:/tmp/spark-85b98228-6260-49b1-b283-09722d64b02f/userFiles-00d1bdcd-e472-4cec-849c-6623ce542e01/test1234.txt does not exist
[2024-12-04T16:36:05.071+0000] {spark_submit.py:634} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 2]
[2024-12-04T16:36:05.071+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO TaskSetManager: Starting task 0.3 in stage 0.0 (TID 3) (172.18.0.7, executor 0, partition 0, PROCESS_LOCAL, 9852 bytes)
[2024-12-04T16:36:05.091+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO TaskSetManager: Lost task 0.3 in stage 0.0 (TID 3) on 172.18.0.7, executor 0: org.apache.spark.SparkFileNotFoundException (File file:/tmp/spark-85b98228-6260-49b1-b283-09722d64b02f/userFiles-00d1bdcd-e472-4cec-849c-6623ce542e01/test1234.txt does not exist
[2024-12-04T16:36:05.093+0000] {spark_submit.py:634} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 3]
[2024-12-04T16:36:05.094+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 ERROR TaskSetManager: Task 0 in stage 0.0 failed 4 times; aborting job
[2024-12-04T16:36:05.094+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO TaskSchedulerImpl: Removed TaskSet 0.0, whose tasks have all completed, from pool
[2024-12-04T16:36:05.095+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO TaskSchedulerImpl: Cancelling stage 0
[2024-12-04T16:36:05.095+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO TaskSchedulerImpl: Killing all running tasks in stage 0: Stage cancelled: Job aborted due to stage failure: Task 0 in stage 0.0 failed 4 times, most recent failure: Lost task 0.3 in stage 0.0 (TID 3) (172.18.0.7 executor 0): org.apache.spark.SparkFileNotFoundException: File file:/tmp/spark-85b98228-6260-49b1-b283-09722d64b02f/userFiles-00d1bdcd-e472-4cec-849c-6623ce542e01/test1234.txt does not exist
[2024-12-04T16:36:05.095+0000] {spark_submit.py:634} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2024-12-04T16:36:05.095+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:781)
[2024-12-04T16:36:05.095+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2024-12-04T16:36:05.096+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2024-12-04T16:36:05.096+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2024-12-04T16:36:05.096+0000] {spark_submit.py:634} INFO - at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2024-12-04T16:36:05.096+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage1.processNext(Unknown Source)
[2024-12-04T16:36:05.096+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2024-12-04T16:36:05.096+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2024-12-04T16:36:05.096+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2024-12-04T16:36:05.097+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-12-04T16:36:05.097+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-12-04T16:36:05.097+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-12-04T16:36:05.097+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-12-04T16:36:05.097+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-12-04T16:36:05.098+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-12-04T16:36:05.098+0000] {spark_submit.py:634} INFO - at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-12-04T16:36:05.098+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-12-04T16:36:05.098+0000] {spark_submit.py:634} INFO - at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-12-04T16:36:05.098+0000] {spark_submit.py:634} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-12-04T16:36:05.099+0000] {spark_submit.py:634} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-12-04T16:36:05.099+0000] {spark_submit.py:634} INFO - at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-12-04T16:36:05.099+0000] {spark_submit.py:634} INFO - at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-12-04T16:36:05.099+0000] {spark_submit.py:634} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2024-12-04T16:36:05.100+0000] {spark_submit.py:634} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2024-12-04T16:36:05.100+0000] {spark_submit.py:634} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-12-04T16:36:05.100+0000] {spark_submit.py:634} INFO - 
[2024-12-04T16:36:05.100+0000] {spark_submit.py:634} INFO - Driver stacktrace:
[2024-12-04T16:36:05.100+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO DAGScheduler: ResultStage 0 (csv at NativeMethodAccessorImpl.java:0) failed in 2.355 s due to Job aborted due to stage failure: Task 0 in stage 0.0 failed 4 times, most recent failure: Lost task 0.3 in stage 0.0 (TID 3) (172.18.0.7 executor 0): org.apache.spark.SparkFileNotFoundException: File file:/tmp/spark-85b98228-6260-49b1-b283-09722d64b02f/userFiles-00d1bdcd-e472-4cec-849c-6623ce542e01/test1234.txt does not exist
[2024-12-04T16:36:05.100+0000] {spark_submit.py:634} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2024-12-04T16:36:05.101+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:781)
[2024-12-04T16:36:05.101+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2024-12-04T16:36:05.101+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2024-12-04T16:36:05.101+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2024-12-04T16:36:05.101+0000] {spark_submit.py:634} INFO - at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2024-12-04T16:36:05.101+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage1.processNext(Unknown Source)
[2024-12-04T16:36:05.102+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2024-12-04T16:36:05.102+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2024-12-04T16:36:05.102+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2024-12-04T16:36:05.102+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-12-04T16:36:05.102+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-12-04T16:36:05.103+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-12-04T16:36:05.103+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-12-04T16:36:05.103+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-12-04T16:36:05.103+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-12-04T16:36:05.103+0000] {spark_submit.py:634} INFO - at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-12-04T16:36:05.103+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-12-04T16:36:05.104+0000] {spark_submit.py:634} INFO - at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-12-04T16:36:05.104+0000] {spark_submit.py:634} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-12-04T16:36:05.104+0000] {spark_submit.py:634} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-12-04T16:36:05.104+0000] {spark_submit.py:634} INFO - at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-12-04T16:36:05.104+0000] {spark_submit.py:634} INFO - at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-12-04T16:36:05.104+0000] {spark_submit.py:634} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2024-12-04T16:36:05.104+0000] {spark_submit.py:634} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2024-12-04T16:36:05.105+0000] {spark_submit.py:634} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-12-04T16:36:05.105+0000] {spark_submit.py:634} INFO - 
[2024-12-04T16:36:05.105+0000] {spark_submit.py:634} INFO - Driver stacktrace:
[2024-12-04T16:36:05.105+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO DAGScheduler: Job 0 failed: csv at NativeMethodAccessorImpl.java:0, took 2.414433 s
[2024-12-04T16:36:05.312+0000] {spark_submit.py:634} INFO - Traceback (most recent call last):
[2024-12-04T16:36:05.315+0000] {spark_submit.py:634} INFO - File "/opt/***/jobs/python/wordcountjob.py", line 46, in <module>
[2024-12-04T16:36:05.317+0000] {spark_submit.py:634} INFO - spark.read.csv(path='file:'+myfile)
[2024-12-04T16:36:05.317+0000] {spark_submit.py:634} INFO - ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-12-04T16:36:05.321+0000] {spark_submit.py:634} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/readwriter.py", line 740, in csv
[2024-12-04T16:36:05.323+0000] {spark_submit.py:634} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1322, in __call__
[2024-12-04T16:36:05.326+0000] {spark_submit.py:634} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/errors/exceptions/captured.py", line 179, in deco
[2024-12-04T16:36:05.327+0000] {spark_submit.py:634} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/protocol.py", line 326, in get_return_value
[2024-12-04T16:36:05.373+0000] {spark_submit.py:634} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling o25.csv.
[2024-12-04T16:36:05.374+0000] {spark_submit.py:634} INFO - : org.apache.spark.SparkException: Job aborted due to stage failure: Task 0 in stage 0.0 failed 4 times, most recent failure: Lost task 0.3 in stage 0.0 (TID 3) (172.18.0.7 executor 0): org.apache.spark.SparkFileNotFoundException: File file:/tmp/spark-85b98228-6260-49b1-b283-09722d64b02f/userFiles-00d1bdcd-e472-4cec-849c-6623ce542e01/test1234.txt does not exist
[2024-12-04T16:36:05.375+0000] {spark_submit.py:634} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2024-12-04T16:36:05.376+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:781)
[2024-12-04T16:36:05.376+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2024-12-04T16:36:05.377+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2024-12-04T16:36:05.377+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2024-12-04T16:36:05.377+0000] {spark_submit.py:634} INFO - at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2024-12-04T16:36:05.380+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage1.processNext(Unknown Source)
[2024-12-04T16:36:05.380+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2024-12-04T16:36:05.380+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2024-12-04T16:36:05.381+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2024-12-04T16:36:05.381+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-12-04T16:36:05.381+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-12-04T16:36:05.382+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-12-04T16:36:05.382+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-12-04T16:36:05.382+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-12-04T16:36:05.382+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-12-04T16:36:05.383+0000] {spark_submit.py:634} INFO - at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-12-04T16:36:05.383+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-12-04T16:36:05.383+0000] {spark_submit.py:634} INFO - at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-12-04T16:36:05.386+0000] {spark_submit.py:634} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-12-04T16:36:05.386+0000] {spark_submit.py:634} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-12-04T16:36:05.387+0000] {spark_submit.py:634} INFO - at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-12-04T16:36:05.387+0000] {spark_submit.py:634} INFO - at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-12-04T16:36:05.387+0000] {spark_submit.py:634} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2024-12-04T16:36:05.388+0000] {spark_submit.py:634} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2024-12-04T16:36:05.389+0000] {spark_submit.py:634} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-12-04T16:36:05.389+0000] {spark_submit.py:634} INFO - 
[2024-12-04T16:36:05.389+0000] {spark_submit.py:634} INFO - Driver stacktrace:
[2024-12-04T16:36:05.389+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.DAGScheduler.failJobAndIndependentStages(DAGScheduler.scala:2856)
[2024-12-04T16:36:05.390+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.DAGScheduler.$anonfun$abortStage$2(DAGScheduler.scala:2792)
[2024-12-04T16:36:05.390+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.DAGScheduler.$anonfun$abortStage$2$adapted(DAGScheduler.scala:2791)
[2024-12-04T16:36:05.392+0000] {spark_submit.py:634} INFO - at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:62)
[2024-12-04T16:36:05.392+0000] {spark_submit.py:634} INFO - at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:55)
[2024-12-04T16:36:05.393+0000] {spark_submit.py:634} INFO - at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:49)
[2024-12-04T16:36:05.393+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.DAGScheduler.abortStage(DAGScheduler.scala:2791)
[2024-12-04T16:36:05.393+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.DAGScheduler.$anonfun$handleTaskSetFailed$1(DAGScheduler.scala:1247)
[2024-12-04T16:36:05.394+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.DAGScheduler.$anonfun$handleTaskSetFailed$1$adapted(DAGScheduler.scala:1247)
[2024-12-04T16:36:05.394+0000] {spark_submit.py:634} INFO - at scala.Option.foreach(Option.scala:407)
[2024-12-04T16:36:05.398+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.DAGScheduler.handleTaskSetFailed(DAGScheduler.scala:1247)
[2024-12-04T16:36:05.398+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.doOnReceive(DAGScheduler.scala:3060)
[2024-12-04T16:36:05.399+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:2994)
[2024-12-04T16:36:05.399+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:2983)
[2024-12-04T16:36:05.399+0000] {spark_submit.py:634} INFO - at org.apache.spark.util.EventLoop$$anon$1.run(EventLoop.scala:49)
[2024-12-04T16:36:05.399+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.DAGScheduler.runJob(DAGScheduler.scala:989)
[2024-12-04T16:36:05.400+0000] {spark_submit.py:634} INFO - at org.apache.spark.SparkContext.runJob(SparkContext.scala:2393)
[2024-12-04T16:36:05.400+0000] {spark_submit.py:634} INFO - at org.apache.spark.SparkContext.runJob(SparkContext.scala:2414)
[2024-12-04T16:36:05.400+0000] {spark_submit.py:634} INFO - at org.apache.spark.SparkContext.runJob(SparkContext.scala:2433)
[2024-12-04T16:36:05.400+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.SparkPlan.executeTake(SparkPlan.scala:530)
[2024-12-04T16:36:05.401+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.SparkPlan.executeTake(SparkPlan.scala:483)
[2024-12-04T16:36:05.401+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.CollectLimitExec.executeCollect(limit.scala:61)
[2024-12-04T16:36:05.401+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.Dataset.collectFromPlan(Dataset.scala:4333)
[2024-12-04T16:36:05.401+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.Dataset.$anonfun$head$1(Dataset.scala:3316)
[2024-12-04T16:36:05.401+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.Dataset.$anonfun$withAction$2(Dataset.scala:4323)
[2024-12-04T16:36:05.402+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.QueryExecution$.withInternalError(QueryExecution.scala:546)
[2024-12-04T16:36:05.402+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.Dataset.$anonfun$withAction$1(Dataset.scala:4321)
[2024-12-04T16:36:05.405+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withNewExecutionId$6(SQLExecution.scala:125)
[2024-12-04T16:36:05.405+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.SQLExecution$.withSQLConfPropagated(SQLExecution.scala:201)
[2024-12-04T16:36:05.405+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withNewExecutionId$1(SQLExecution.scala:108)
[2024-12-04T16:36:05.405+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.SparkSession.withActive(SparkSession.scala:900)
[2024-12-04T16:36:05.406+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.SQLExecution$.withNewExecutionId(SQLExecution.scala:66)
[2024-12-04T16:36:05.406+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.Dataset.withAction(Dataset.scala:4321)
[2024-12-04T16:36:05.406+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.Dataset.head(Dataset.scala:3316)
[2024-12-04T16:36:05.406+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.Dataset.take(Dataset.scala:3539)
[2024-12-04T16:36:05.406+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.csv.TextInputCSVDataSource$.infer(CSVDataSource.scala:111)
[2024-12-04T16:36:05.407+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.csv.CSVDataSource.inferSchema(CSVDataSource.scala:64)
[2024-12-04T16:36:05.407+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.csv.CSVFileFormat.inferSchema(CSVFileFormat.scala:62)
[2024-12-04T16:36:05.407+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.DataSource.$anonfun$getOrInferFileFormatSchema$11(DataSource.scala:208)
[2024-12-04T16:36:05.407+0000] {spark_submit.py:634} INFO - at scala.Option.orElse(Option.scala:447)
[2024-12-04T16:36:05.407+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.DataSource.getOrInferFileFormatSchema(DataSource.scala:205)
[2024-12-04T16:36:05.408+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.DataSource.resolveRelation(DataSource.scala:407)
[2024-12-04T16:36:05.408+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.DataFrameReader.loadV1Source(DataFrameReader.scala:229)
[2024-12-04T16:36:05.408+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.DataFrameReader.$anonfun$load$2(DataFrameReader.scala:211)
[2024-12-04T16:36:05.409+0000] {spark_submit.py:634} INFO - at scala.Option.getOrElse(Option.scala:189)
[2024-12-04T16:36:05.409+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.DataFrameReader.load(DataFrameReader.scala:211)
[2024-12-04T16:36:05.411+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.DataFrameReader.csv(DataFrameReader.scala:538)
[2024-12-04T16:36:05.412+0000] {spark_submit.py:634} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-12-04T16:36:05.412+0000] {spark_submit.py:634} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[2024-12-04T16:36:05.412+0000] {spark_submit.py:634} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-12-04T16:36:05.412+0000] {spark_submit.py:634} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:569)
[2024-12-04T16:36:05.413+0000] {spark_submit.py:634} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2024-12-04T16:36:05.413+0000] {spark_submit.py:634} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-12-04T16:36:05.413+0000] {spark_submit.py:634} INFO - at py4j.Gateway.invoke(Gateway.java:282)
[2024-12-04T16:36:05.416+0000] {spark_submit.py:634} INFO - at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2024-12-04T16:36:05.416+0000] {spark_submit.py:634} INFO - at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2024-12-04T16:36:05.416+0000] {spark_submit.py:634} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-12-04T16:36:05.417+0000] {spark_submit.py:634} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-12-04T16:36:05.417+0000] {spark_submit.py:634} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-12-04T16:36:05.417+0000] {spark_submit.py:634} INFO - Caused by: org.apache.spark.SparkFileNotFoundException: File file:/tmp/spark-85b98228-6260-49b1-b283-09722d64b02f/userFiles-00d1bdcd-e472-4cec-849c-6623ce542e01/test1234.txt does not exist
[2024-12-04T16:36:05.417+0000] {spark_submit.py:634} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2024-12-04T16:36:05.417+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:781)
[2024-12-04T16:36:05.417+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2024-12-04T16:36:05.418+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2024-12-04T16:36:05.418+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2024-12-04T16:36:05.418+0000] {spark_submit.py:634} INFO - at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2024-12-04T16:36:05.418+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage1.processNext(Unknown Source)
[2024-12-04T16:36:05.418+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2024-12-04T16:36:05.418+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2024-12-04T16:36:05.418+0000] {spark_submit.py:634} INFO - at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2024-12-04T16:36:05.419+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-12-04T16:36:05.419+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-12-04T16:36:05.419+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-12-04T16:36:05.419+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-12-04T16:36:05.419+0000] {spark_submit.py:634} INFO - at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-12-04T16:36:05.419+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-12-04T16:36:05.419+0000] {spark_submit.py:634} INFO - at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-12-04T16:36:05.420+0000] {spark_submit.py:634} INFO - at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-12-04T16:36:05.421+0000] {spark_submit.py:634} INFO - at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-12-04T16:36:05.423+0000] {spark_submit.py:634} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-12-04T16:36:05.423+0000] {spark_submit.py:634} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-12-04T16:36:05.424+0000] {spark_submit.py:634} INFO - at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-12-04T16:36:05.424+0000] {spark_submit.py:634} INFO - at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-12-04T16:36:05.424+0000] {spark_submit.py:634} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2024-12-04T16:36:05.424+0000] {spark_submit.py:634} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2024-12-04T16:36:05.424+0000] {spark_submit.py:634} INFO - ... 1 more
[2024-12-04T16:36:05.424+0000] {spark_submit.py:634} INFO - 
[2024-12-04T16:36:05.489+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO SparkContext: Invoking stop() from shutdown hook
[2024-12-04T16:36:05.489+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-12-04T16:36:05.506+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO SparkUI: Stopped Spark web UI at http://2a892a3469ec:4042
[2024-12-04T16:36:05.513+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO StandaloneSchedulerBackend: Shutting down all executors
[2024-12-04T16:36:05.514+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Asking each executor to shut down
[2024-12-04T16:36:05.550+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-12-04T16:36:05.615+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO MemoryStore: MemoryStore cleared
[2024-12-04T16:36:05.619+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 WARN Dispatcher: Message RemoteProcessDisconnected(172.18.0.7:58862) dropped. Could not find BlockManagerEndpoint1.
[2024-12-04T16:36:05.622+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO BlockManager: BlockManager stopped
[2024-12-04T16:36:05.635+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-12-04T16:36:05.638+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-12-04T16:36:05.663+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO SparkContext: Successfully stopped SparkContext
[2024-12-04T16:36:05.664+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO ShutdownHookManager: Shutdown hook called
[2024-12-04T16:36:05.666+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO ShutdownHookManager: Deleting directory /tmp/spark-85b98228-6260-49b1-b283-09722d64b02f
[2024-12-04T16:36:05.673+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO ShutdownHookManager: Deleting directory /tmp/spark-85b98228-6260-49b1-b283-09722d64b02f/pyspark-529472dd-3044-4f85-aadd-2a7a04cf65d6
[2024-12-04T16:36:05.680+0000] {spark_submit.py:634} INFO - 24/12/04 16:36:05 INFO ShutdownHookManager: Deleting directory /tmp/spark-526b0344-84d7-4f30-a4ef-96f3b0906faa
[2024-12-04T16:36:05.764+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 176, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 560, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://sparkingflow-spark-master-1:7077 --name arrow-spark --deploy-mode client jobs/python/wordcountjob.py. Error code is: 1.
[2024-12-04T16:36:05.776+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=sparking_flow, task_id=python_job, run_id=manual__2024-12-04T16:35:45.230775+00:00, execution_date=20241204T163545, start_date=20241204T163547, end_date=20241204T163605
[2024-12-04T16:36:05.797+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-04T16:36:05.797+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 89 for task python_job (Cannot execute: spark-submit --master spark://sparkingflow-spark-master-1:7077 --name arrow-spark --deploy-mode client jobs/python/wordcountjob.py. Error code is: 1.; 189)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 176, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 560, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://sparkingflow-spark-master-1:7077 --name arrow-spark --deploy-mode client jobs/python/wordcountjob.py. Error code is: 1.
[2024-12-04T16:36:05.847+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-12-04T16:36:05.885+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-04T16:36:05.891+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
